contrib/fontconfig/P01-mingw-msys-ln.patch

```diff
@@ -0,0 +1,11 @@
+--- ./Makefile.am.orig	2017-01-10 01:00:19.634549000 +0100
```

Comment:
We'll be able to remove this as soon as https://github.com/HandBrake/HandBrake/pull/610 is merged